tlg-base-config_3-vm/nested/addc.json

```diff
@@ -0,0 +1,183 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string"
+    },
+    "adVMName": {
+      "type": "string"
+    },
+    "adVMSize": {
+      "type": "string"
+    },
+    "adminUsername": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "storageAccountNamePrefix": {
+      "type": "string"
+    },
+    "adNicName": {
+      "type": "string"
+    },
+    "adModulesURL": {
+      "type": "string"
+    },
+    "adConfigurationFunction": {
+      "type": "string"
+    },
+    "SasToken": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-06-15",
```

Comment:
Done.

Reply:
According to VSCode, 2015-06-15 is the latest available apiVersion. Am I missing something? 
Also, in one of my last commits, I revised the way I pass params to nested templates to use a SharedVmConfig object instead of passing the same individual params to all nested templates, but that's not showing up here. Perhaps my next commit will resolve that.